packages/office-ui-fabric-react/src/components/Modal/examples/Modal.TopMost.Example.tsx

```diff
@@ -0,0 +1,76 @@
+import { DefaultButton } from 'office-ui-fabric-react/lib/Button';
+import { getId } from 'office-ui-fabric-react/lib/Utilities';
+import * as styles from './Modal.Example.scss';
+import { Checkbox } from 'office-ui-fabric-react/lib/Checkbox';
+import { ContextualMenu } from 'office-ui-fabric-react/lib/ContextualMenu';
+
+export interface IModalModelessExampleState {
+  showModal: boolean;
+  isDraggable: boolean;
+}
+
+export class ModalTopMostExample extends React.Component<{}, IModalModelessExampleState> {
+  public state: IModalModelessExampleState = {
+    showModal: false,
+    isDraggable: false
+  };
+  // Use getId() to ensure that the IDs are unique on the page.
+  // (It's also okay to use plain strings without getId() and manually ensure uniqueness.)
+  private _titleId: string = getId('title');
+  private _subtitleId: string = getId('subText');
+  private _dragOptions: IDragOptions = {
+    moveMenuItemText: 'Move',
+    closeMenuItemText: 'Close',
+    menu: ContextualMenu
+  };
+
+  public render(): JSX.Element {
+    const { showModal, isDraggable } = this.state;
+    return (
+      <div>
+        <Checkbox label="Is draggable" onChange={this._toggleDraggable} checked={isDraggable} disabled={showModal} />
+        <DefaultButton secondaryText="Opens the Sample Modal" onClick={this._showModal} text="Open Modal" />
+        <Modal
+          titleAriaId={this._titleId}
+          subtitleAriaId={this._subtitleId}
+          isOpen={showModal}
+          onDismiss={this._closeModal}
+          isModeless={true}
+          containerClassName={styles.stickyContainer}
+          dragOptions={isDraggable ? this._dragOptions : undefined}
+          layerProps={{ topMostDocument: true }}
+        >
+          <div className={styles.header}>
+            <span id={this._titleId}>Lorem Ipsum</span>
+          </div>
+          <div id={this._subtitleId} className={styles.body}>
+            <DefaultButton onClick={this._closeModal} text="Close" />
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas lorem nulla, malesuada ut sagittis sit amet, vulputate in
+              leo. Maecenas vulputate congue sapien eu tincidunt. Etiam eu sem turpis. Fusce tempor sagittis nunc, ut interdum ipsum
+              vestibulum non. Proin dolor elit, aliquam eget tincidunt non, vestibulum ut turpis. In hac habitasse platea dictumst. In a
+              odio eget enim porttitor maximus. Aliquam nulla nibh, ullamcorper aliquam placerat eu, viverra et dui. Phasellus ex lectus,
+              maximus in mollis ac, luctus vel eros. Vivamus ultrices, turpis sed malesuada gravida, eros ipsum venenatis elit, et volutpat
+              eros dui et ante. Quisque ultricies mi nec leo ultricies mollis. Vivamus egestas volutpat lacinia. Quisque pharetra eleifend
+              efficitur.{' '}
+            </p>
+          </div>
+        </Modal>
+      </div>
```

Comment:
It's hard to tell how similar this example is compared to any others. Is it possible to use an `iframe` to demo how this would be used, or is this example already doing that somehow?

Reply:
It was still work in progress at the time, but I was initially planning on using an iframe in the demo. 
I removed it now, since I don't think we should put all of that into the example just to test one iframe. It would also require us to load all the scripts, etc. again in the iframe.

Do you see any other way of showcasing / testing this properly?

Reply:
I'm actually curious to see the difference myself since I can't visualize it. 😄 How about modifying or adding a Toggle to an existing example that adds the prop and an iframe wrapper somehow?

Reply:
Hmm, I do get your point about the iframe. I just don't know this feature well enough to propose a meaningful alternative.

Reply:
I'm not quite sure how to showcase this too at the moment.
I think we need another page we can show in the iframe and then make sure that when rendering something in the iframe with the `renderAtRoot` set to true, it'll be displayed in the parent (root) document.
I'll experiment a bit with it and let you know, if I could come up with a clean solution.